### PR TITLE
SPM/secure_services: fix printing

### PR DESCRIPTION
**Pull request by oyvindronningstad**
Cosmetic fixes.

### CODE REVIEW

**Comment by oyvindronningstad** [comment]
cc @hakonfam

File: samples/nrf9160/secure_services/src/main.c [hunk 1]
```diff
@@ -48,18 +55,17 @@ void main(void)
...
 	}
-	for (int i = 0; i < num_bytes_to_read; i++) {
-		printk("%d", buf[i]);
-	}
+
+	print_large_hex_string(buf, num_bytes_to_read);
 #endif
 
-	printk("\nReboot in %d ms.\n", sleep_time_ms);
+	printk("\nReboot in %d seconds.\n", sleep_time_ms/1000);
```
[the remark this answers is not in the document]
Comment (oyvindronningstad) [reply samples/nrf9160/secure_services/src/main.c]: Thanks, added.

**Comment by oyvindronningstad** [comment]
@lemrey merge?

**Comment by ioannisg** [comment]
Can merge, but want @lemrey to approve it :)

**Comment by lemrey** [comment]
We have fixed the compilation errors; this sample was never affected, in fact is one of the few that is built.

**Pull request merged**